dotnet/samples/06.auth.oauth.bot/Program.cs

```diff
@@ -44,6 +44,7 @@
         Title = "Sign In",
         Text = "Please sign in to use the bot.",
         EndOnInvalidMessage = true,
+        EnableSso = true, // Set this to true to enable SSO
```

Comment:
this comments seems unnecessary given the variable name is pretty self explanatory